### ISSUE TITLE
silverware upright (suggesting -on)

### ISSUE THREAD

**Issue by hvbhome** [Templarian/MaterialDesign#3204]
please consider adding a mdi:silverware version with the fork and knife (or spoon) in parallel upright position .
it would accommodate an -off and -on situation: crossed for off, parallel for on ;-)
would be cool indeed if possible ,
thanks
Marius

**Comment by GreenTurtwig** [comment]
This is `restaurant` in the stock set, should probably get it added as a variant to `silverware-variant`, although that's already a variant itself...
Renaming `silverware-variant` to `knife-and-fork` and adding it as `knife-and-fork-variant` could be a solution.

**Comment by hvbhome** [comment]
would be appreciated indeed.thx

**Comment by PeterShaggyNoble** [comment]
Duplicate of #1802
